app/config.template.json

```diff
@@ -23,12 +23,12 @@
     },
     "SqlSettings": {
         "DriverName": "postgres",
-        "DataSource": "postgres://MM_USERNAME:MM_PASSWORD@DB_HOST:DB_PORT/MM_DBNAME?sslmode=disable&connect_timeout=10",
+        "DataSource": "postgres://mattermost:POSTGRES_PASSWORD@db:5432/mattermost?sslmode=disable&connect_timeout=10",
```

Comment:
is there any reason to hard code the those values? Again, this may break if people don't use docker-compose. Or they have a non-docker based db that has some restriction on username or db name.